packages/animations/browser/src/dsl/animation_ast_builder.ts

```diff
@@ -75,8 +75,8 @@ export class AnimationAstBuilderVisitor implements AnimationDslVisitor {
 
   private _resetContextStyleTimingState(context: AnimationAstBuilderContext) {
     context.currentQuerySelector = ROOT_SELECTOR;
-    context.collectedStyles = {};
-    context.collectedStyles[ROOT_SELECTOR] = {};
+    context.collectedStyles = new Map<string, Map<string, StyleTimeTuple>>();
+    context.collectedStyles.set(ROOT_SELECTOR, new Map<string, StyleTimeTuple>());
```

Comment:
I haven't checked, but I guess we don't need an extra type annotation here, it'd probably be inferred.
```suggestion
    context.collectedStyles.set(ROOT_SELECTOR, new Map());
```